data/remnant ships.txt

```diff
@@ -413,6 +493,62 @@ ship "Pelican"
 
 
 
+ship "Petrel"
+	sprite "ship/petrel"
+	thumbnail "thumbnail/petrel"
+	attributes
+		category "Fighter"
+		licenses
+			Remnant
+		"cost" 4500000
+		"shields" 2500
+		"hull" 3000
+		"cargo space" 20
+		"fuel capacity" 100
+		"ramscoop" 1
+		"required crew" 1
+		"bunks" 3
+		"mass" 35
+		"drag" .9
+		"heat dissipation" .9
+		"outfit space" 195
+		"shield generation" 1.9
+		"shield energy" 1.1
+		"hull repair rate" 0.9
+		"hull energy" 0.6
+		"outfit scan power" 12
+		"outfit scan speed" 1
+		"tactical scan power" 26
+		"asteroid scan power" 30
+		"weapon capacity" 80
+		"engine capacity" 55
+		weapon
+			"blast radius" 15
+			"shield damage" 150
+			"hull damage" 200
+			"hit force" 350
+	outfits
+		"Inhibitor Cannon" 3
+		
+		"Millennium Cell" 2
+		"Crystal Capacitor"
+		"Thermoelectric Cooler" 3
+		
+		"Crucible-Class Thruster"
+		"Crucible-Class Steering"
+		
+	gun 0 -38.5 "Inhibitor Cannon"
+	gun -10.5 -24 "Inhibitor Cannon"
+	gun 10.5 -24 "Inhibitor Cannon"
+	turret 0 2.5
+	engine 0 41.5
+	explode "tiny explosion" 30
+	explode "small explosion" 20
+	description "The Petrel is a multi-role mining/wing leader fightercraft designed to give remnant a forward control position for flights of Terns in higher risk or quickly changing environments. Their initial design was used as a urban mobile gun platform that flitted around structures, caverns, and natural terrain on their worlds while fighting on-planet threats."
+	description "Due to the comparatively small pool the Remnant draw on for pilots and crew they are loath to throw mass waves of people into the line of fire. As such, their fighters are not intended to be up-close glass cannons like most other factions of humanity tend to build. Instead, these ships are highly agile fighters equipped with medium range weaponry designed to snipe enemies from behind cover."
```

Comment:
Multi-paragraph descriptions should have every paragraph after the first indented with a tab.

Reply:
Fixed. Thanks!